lib/aramanim.py

```diff
@@ -193,3 +193,17 @@ def interpolate(self, alpha):
 
 def get_segment_part(length, label=None, color=WHITE, **kwargs):
     return Segment(ORIGIN, length * RIGHT, label, stroke_width=6, color=color, **kwargs)
+
+
+class ConectionLine(VMobject):
+    """
+    Draws lines under two members of a formula. Usually used for explaining parentheses rules
```

Comment:
Does this draw a separate underline for two objects?
Can you show an example?

Reply:
Սա ինձ @a-arakelian-ն ա բացատրել։ Էս ֆունկցիան 3 կետեր ա դնում, ու որպես բեկյալ գծերով միացնում իրար։ Պռակտիկությունից ելնելով, կարելի ա ասել Draws a segment between two members of a formula, եթե էդպես ավելի հասկանալի ա հնչում։ Օրինակ՝ https://youtu.be/qPZ7_xQ7kVI?t=238